controller/Lists.php

```diff
@@ -149,9 +319,19 @@ public function getListElements()
         if ($this->hasRequestParameter('listUri')) {
             $list = $this->getListService()->getList(tao_helpers_Uri::decode($this->getRequestParameter('listUri')));
             if (!is_null($list)) {
-                foreach ($this->getListService()->getListELements($list, true) as $listElement) {
+                $isRemote = $this->getListService()->isRemote($list);
+
+                $limit    = $isRemote
+                    ? self::REMOTE_LIST_PREVIEW_LIMIT
+                    : 0;
+
+                foreach ($this->getListService()->getListELements($list, true, $limit) as $listElement) {
                     $data[tao_helpers_Uri::encode($listElement->getUri())] = $listElement->getLabel();
                 }
+
+                if ($isRemote && count($data) === self::REMOTE_LIST_PREVIEW_LIMIT) {
+                    $data[''] = '...';
```

Comment:
Is the key missed here?

Reply:
No, it doesn't have any meaning in the context, but all the other keys are string values.

Reply:
Yes, about other keys it is clear, but what the reason to add value under empty key?

Reply:
That's just a quick hack to highlight the fact the list, being shown, is incomplete, and there's more to it.
![image](https://user-images.githubusercontent.com/2943256/87686133-7160c800-c784-11ea-9626-85265de279a6.png)